packages/andromeda-socket/src/astroport.rs

```diff
@@ -317,3 +332,36 @@ pub fn transform_asset_info(
         }),
     }
 }
+
+/// Transforms a single AndromedaAssetEntry into AssetInfo
+pub fn transform_asset_entry_info(
```

Comment:
This function and `transform_asset_info` can be deleted and just make `transform_asset_info_core` a public function. And then rename it to `transform_asset_info`